src/ncp-spinel/SpinelNCPInstance.h

```diff
@@ -241,6 +241,9 @@ class SpinelNCPInstance : public NCPInstanceBase {
 
 	int mSubPTIndex;
 
+	uint16_t mPanid;
+	uint64_t mXPanid;
```

Comment:
These are redundant. These fields are already present in `NCPInstanceBase:: mCurrentNetworkInstance`.